ops/ansible/playbooks-ccs/vars/prod-sbx/group_vars/all/ssh.yml

```diff
@@ -1,498 +1,458 @@
+66313463623963363863386163366133653232643864653537323162626262363930656639393165
+39326634316133353434656139616563663735383236333964346530636437323737636235313631
+36326162626434383036373235353230336136653331353832393536306165313537666365346536
+65626238623037356134626130306235326437616435323866343431393739336165623361383861
+35316665653935623732376134313139396531643436616365323830626133636436613739313265
+38336632356663373233376438303964366666623135336230316265613035383566313433303232
+61663932343336346331623436633135343630666139623466656663316263373963383139313237
+62616239343739323439633634363935363165613361383765383037393634363832303234353634
+33393234633839646562396338326131623034633235346639353762323462626366393434353865
+35623035323964373365613738353466616161633665643361303035343964666439653630303034
+37353938303731303834393533646339393032316163376161336263313265326238376264316166
+36363935343866613831633733326631633733393333653833306463323133313166333461623166
+32663234383163643231353935613766666332363331653831323535373665363233396634353230
+37666135636430343133653637366366663936323239333765386335386230383135313134313962
+65313461303233366137653735366133643531373736393234343962653239656439633735366662
+35376438333439353663636237396438663830623834333466353666623431666362666564623332
+32353432373736613162653765353833363366336664333034633762333237346562633434363961
+36633230363533323661646663366437666636303433306638323261626262336463336263326464
+30616366316637323763623830366331383061333734383237613766386262666230393961646365
+36363032616539313539353063636465343564333833383234663562343832656333363334626539
+66626136303362353836333030316564646362396534613233366238373132323235303636643036
+34616463373261303864623163316330373163313236643535313264353438363664343436336136
+66643330376466653837373936663837613239636465363664613230363239393834303931326232
+39623931363332623735346130636533386264623933343166306564663233363737326235303932
+31336232643361646462313630343663383936333863373161373063396666623532666364613834
+39383837376665346230333566663735383566393130616538326161326465636437303962663664
+30323365663333323831383430366131313032356164343530363762626532363139623737633833
+63666265613864373437633766363933393965613232323261616663616261373163656637646561
+64343836316333646433373136613537363462333934366632653436363866343163393731313862
+31306436313037373439316166393531353462333061363466613464616162666362323837383834
+62616366323764323762613962663634363561313439326333626137633662336338366532646438
+64343232666231643165306235323634653732333638666138353836383431396363366163376132
+63326365363130653633323239336130373436346663313539353238396532633733303832306464
+61636465663665633935646530393231656635316565303166373037353533383338396636306562
+36653833313064316465646135343437373534386239386233393761626634383837626264346133
+63663731396262336262616666353563663934386136373137363565373238653038
```

Comment:
What were the ansible changes here about?

Reply:
merged master into ccw branch to bring it up to date